packages/css/card.css

```diff
@@ -13,178 +13,176 @@
-  color: inherit;
-}
-
-.ds-card--link h1,
-.ds-card--link h2,
-.ds-card--link h3,
-.ds-card--link h4,
-.ds-card--link h5,
-.ds-card--link h6 {
-  text-decoration: underline;
-  text-decoration-thickness: max(1px, 0.0625rem, 0.1025em);
-  text-underline-offset: max(6px, 0.25rem, 0.22em);
-}
-
-.ds-card--neutral {
-  --dsc-card-border-color: var(--ds-color-neutral-border-subtle);
-  --dsc-card-backround: var(--ds-color-neutral-background-default);
-}
-
-.ds-card--neutral.ds-card--link:hover {
-  --dsc-card-border-color: var(--ds-color-neutral-border-subtle);
-  --dsc-card-backround: var(--ds-color-neutral-background-subtle);
-}
-
-.ds-card--neutral.ds-card--link:active {
-  --dsc-card-backround: var(--ds-color-neutral-surface-default);
-}
-
-.ds-card--subtle {
-  --dsc-card-border-color: var(--ds-color-neutral-border-subtle);
-  --dsc-card-backround: var(--ds-color-neutral-background-subtle);
-}
-
-.ds-card--subtle.ds-card--link:hover {
-  --dsc-card-border-color: var(--ds-color-neutral-border-default);
-  --dsc-card-backround: var(--ds-color-neutral-surface-default);
-}
-
-.ds-card--subtle.ds-card--link:active {
-  --dsc-card-backround: var(--ds-color-neutral-surface-hover);
-}
-
-.ds-card--brand1 {
-  --dsc-card-border-color: var(--ds-color-brand1-border-subtle);
-  --dsc-card-backround: var(--ds-color-brand1-surface-default);
-  --dsc-card-color: var(--ds-color-brand1-text-default);
-}
-
-.ds-card--brand1.ds-card--link {
-  --dsc-card-border-color: var(--ds-color-brand1-border-default);
-}
-
-.ds-card--brand1.ds-card--link:hover {
-  --dsc-card-backround: var(--ds-color-brand1-surface-hover);
-}
-
-.ds-card--brand1.ds-card--link:active {
-  --dsc-card-backround: var(--ds-color-brand1-surface-active);
-}
-
-.ds-card--brand2 {
-  --dsc-card-border-color: var(--ds-color-brand2-border-subtle);
-  --dsc-card-backround: var(--ds-color-brand2-surface-default);
-  --dsc-card-color: var(--ds-color-brand2-text-default);
-}
-
-.ds-card--brand2.ds-card--link {
-  --dsc-card-border-color: var(--ds-color-brand2-border-default);
-}
-
-.ds-card--brand2.ds-card--link:hover {
-  --dsc-card-backround: var(--ds-color-brand2-surface-hover);
-}
-
-.ds-card--brand2.ds-card--link:active {
-  --dsc-card-backround: var(--ds-color-brand2-surface-active);
-}
-
-.ds-card--brand3 {
-  --dsc-card-border-color: var(--ds-color-brand3-border-subtle);
-  --dsc-card-backround: var(--ds-color-brand3-surface-default);
-  --dsc-card-color: var(--ds-color-brand3-text-default);
-}
-
-.ds-card--brand3.ds-card--link {
-  --dsc-card-border-color: var(--ds-color-brand3-border-default);
-}
-
-.ds-card--brand3.ds-card--link:hover {
-  --dsc-card-backround: var(--ds-color-brand3-surface-hover);
-}
 
-.ds-card--brand3.ds-card--link:active {
-  --dsc-card-backround: var(--ds-color-brand3-surface-active);
+  &[data-link] {
+    --dsc-card-border-color: var(--ds-color-neutral-border-default);
+
+    text-decoration: none;
+
+    & h1,
+    & h2,
+    & h3,
+    & h4,
+    & h5,
+    & h6 {
```

Comment:
Skulle vi heller gjort `& :is(h1,h2,h3,h4,h5,h6)`? 😊 
og hvorfor setter vi `color` på heading - arver ikke den farge? 😇

Reply:
Me setter farge slik at den blir konsekvent rett i forhold til fargen kortet har. Jo den arver farge, men vår `Heading` komponent setter også farge til `--ds-color-neutral-text-default`, som då blir satt om me ikkje setter farge her.

Reply:
Ah, hvorfor setter Heading farge? Kan det være litt vanskelig å sette en heading på en flate også arver den ikke currentcolor? Burde vi snakke med design om dette? :)

Reply:
Gjerne ta det med design, føler ikkje det er eit valg me i kode skal ta.

Reply:
Tenker det har litt med kode å gjøre fordi det setter føringer for å kunne gjenbrukbare komponenter inni flater - jeg poster en diskusjon om dette her: https://designsystemet.slack.com/archives/C05U1MNKYCX/p1726480465469179

Reply:
Okei. Avventer endringa til me får svar der.

Reply:
Tenker vi kan endre på det senere - kanskje bare merge denne nå først så lager jeg eget issue med color: inherit på typografi-komponenter ☺️